analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.Branching.cs

```diff
@@ -63,22 +63,22 @@ public void Branching_BlockProcessingOrder_CS()
     Tag(""Else"");
 }
 Tag(""End"");";
-            SETestContext.CreateCS(code).Validator.ValidateTagOrder(
-                "Entry",
-                "BeforeTry",        // Dequeue for "if" branch
-                "Else",             // Dequeue for "else" branch
-                "InTry",            // Dequeue after "if" branch
-                "End",              // Dequeue after "else" branch, reaching exit block
-                "InCatch",          // Dequeue after the "try body" that could throw
-                "InFinally",        // Dequeue after the "try body"
-                "InFinally",        // Dequeue after the "catch", with Exception
-                "AfterFinally");    // Dequeue after "if" branch
-        }
+        SETestContext.CreateCS(code).Validator.ValidateTagOrder(
+            "Entry",
+            "BeforeTry",        // Dequeue for "if" branch
+            "Else",             // Dequeue for "else" branch
+            "InTry",            // Dequeue after "if" branch
+            "End",              // Dequeue after "else" branch, reaching exit block
+            "InCatch",          // Dequeue after the "try body" that could throw
+            "InFinally",        // Dequeue after the "try body"
+            "InFinally",        // Dequeue after the "catch", with Exception
+            "AfterFinally");    // Dequeue after "if" branch
+    }
 
-        [TestMethod]
-        public void Branching_BlockProcessingOrder_VB()
-        {
-            const string code = @"
+    [TestMethod]
+    public void Branching_BlockProcessingOrder_Property_VB()
+    {
+        const string code = @"
 Tag(""Entry"")
```

Comment:
It would be good to use `"""` here too: see the other comment.

Reply:
Same as above